src/org/openstreetmap/josm/gui/dialogs/properties/TagEditHelper.java

```diff
@@ -243,6 +243,18 @@ public final String getDataKey(int viewRow) {
         return tagData.getValueAt(tagTable.convertRowIndexToModel(viewRow), 0).toString();
     }
 
+    /**
+     * Finds the keys for given rows of tag editor.
+     * @param viewRows index of rows
+     * @return keys of tags
+     * @since xxx
+     */
+    public final List<String> getDataKeys(int[] viewRows) {
+        return Arrays.stream(viewRows).boxed()
+            .map(r -> tagData.getValueAt(tagTable.convertRowIndexToModel(r), 0).toString())
+            .collect(Collectors.toList());
```

Comment:
I don't think you need to do boxing here.
```suggestion
        return Arrays.stream(viewRows).map(tagTable::convertRowIndexToModel)
            .mapToObj(r -> tagData.getValueAt(r, 0).toString()).collect(Collectors.toList());
```